tests/AutoReview/FixerTest.php

```diff
@@ -176,17 +177,17 @@ public function testDeprecatedFixersHaveCorrectSummary(FixerInterface $fixer)
         $reflection = new \ReflectionClass($fixer);
         $comment = $reflection->getDocComment();
 
+        $this->assertNotRegExp(
+            '/DEPRECATED/',
+            $fixer->getDefinition()->getSummary(),
+            'Fixer cannot contain word "DEPRECATED" in summary'
+        );
+
         if (is_string($comment) && false !== strpos($comment, '@deprecated')) {
-            $this->assertRegExp(
-                '/\. DEPRECATED: use `[a-z_]+` instead\.$/',
-                $fixer->getDefinition()->getSummary(),
-                'Deprecated fixer must contain correct "DEPRECATED" note in summary'
-            );
-        } else {
-            $this->assertNotRegExp(
-                '/DEPRECATED/',
-                $fixer->getDefinition()->getSummary(),
-                'Non-deprecated fixer cannot contain word "DEPRECATED" in summary'
+            $this->assertInstanceOf(
```

Comment:
I would reverse the condition: if a fixer implements the `DeprecatedFixerInterface`, it must have the `@deprecated` annotation, otherwise it must not.